spec/features/users_can_view_reports_spec.rb

```diff
@@ -132,25 +132,43 @@ def expect_to_see_grouped_rows_of_reports_for_an_organisation(organisation:, exp
       expect(page).to have_content report.fund.source_fund.name
     end
 
-    scenario "the report includes a summary" do
-      report = create(:report, :active, organisation: build(:partner_organisation))
+    describe "the report summary" do
+      context "when the fund is ISPF" do
+        scenario "the report includes a summary" do
+          report = create(:report, :for_ispf, :active, organisation: build(:partner_organisation), is_oda: true)
 
-      visit reports_path
+          visit reports_path
 
-      within "##{report.id}" do
-        click_on t("default.link.show")
+          within "##{report.id}" do
+            click_on t("default.link.show")
+          end
+
+          expect(page).to have_content report.description
+          expect(page).to have_content l(report.deadline)
+          within("#is-oda > dd") do
+            expect(page).to have_content "ODA"
+          end
+          expect(page).not_to have_content t("page_content.report.summary.editable.#{report.editable?}")
+          expect(page).to have_content report.organisation.name
+
+          expect(page).to have_content t("page_content.tab_content.summary.activities_added")
+          expect(page).to have_content t("page_content.tab_content.summary.activities_updated")
+          expect(page).to have_content t("page_content.tab_content.summary.actuals_total")
+          expect(page).to have_content t("page_content.tab_content.summary.forecasts_total")
+          expect(page).to have_content t("page_content.tab_content.summary.refunds_total")
+        end
       end
 
-      expect(page).to have_content report.description
-      expect(page).to have_content l(report.deadline)
-      expect(page).not_to have_content t("page_content.report.summary.editable.#{report.editable?}")
-      expect(page).to have_content report.organisation.name
+      context "when the fund is any other" do
+        scenario "the report summary does not include the ODA state" do
```

Comment:
This might be a preference thing, but it doesn't feel right that we only test the negatives for a generic, non-ISPF fund.

Usually the way we test is the generic case first, with all the trimmings, and then add a test for the special cases such as ISPF. What we want to communicate through our tests is "the reports always have a summary with all these things, and then there's also ISPF which has an extra field".

In this case we do want to test that non-ISPF funds don't accidentally show the ODA row, but negative-only tests have a higher risk of passing by accident. (For example, we break displays for non-ISPF funds, and the page doesn't show a summary at all. The test would still pass.)